astro_2.0/components/AppHeader/components/NotificationsBell/tests/__snapshots__/NotificationsBell.spec.tsx.snap

```diff
@@ -20,4 +20,20 @@ exports[`notifications bell Should render component if feature flag is true and
 
 exports[`notifications bell Should render nothing if feature flag is false 1`] = `<div />`;
 
-exports[`notifications bell Should render nothing if user is not logged in 1`] = `<div />`;
+exports[`notifications bell Should render nothing if user is not logged in 1`] = `
```

Comment:
Please, get latest develop. Result of test is incorrect and test is redundant itself.